src/components/ArticleComplexCard.jsx

```diff
@@ -95,27 +114,30 @@ export default function ArticleCard({article, users}) {
         component="img"
         height="194"
         image={`${article.article_img_url}`}
-        // alt="Paella dish"
+        alt="Article image"
       />
       <CardContent>
         <Typography variant="h6" color="text.primary">
         {`${article.title}`}
         </Typography>
       </CardContent>
       <CardActions disableSpacing>
-        <IconButton aria-label="up vote article">
-          <ThumbUpAltOutlinedIcon />
+        <IconButton aria-label="up vote article" onClick={handleArticleVote}>
+          <ThumbUpAltOutlinedIcon color={ hasVoted ? `disabled` : 'enabled'}/>
         </IconButton>
-        <Chip label={`Votes: ${fullArticle.votes}`} variant="outlined" />
+        <Chip label={`Votes: ${articleVotes}`} variant="outlined" />
+        <Typography variant='h6' sx={{marginLeft: 'auto'}}>Read</Typography>
```

Comment:
There's. lot of header stepping in your app. Remember, we should use headings in their numerical order, even if we have more than one of a type